src/test/java/de/retest/recheck/ui/diff/AlignmentTest.java

```diff
@@ -144,6 +154,51 @@ public void alignment_with_less_parents() throws Exception {
 		assertThat( alignment.get( expA0 ) ).isSameAs( actA );
 	}
 
+	@Test
+	public void very_different_elements_should_not_be_aligned() {
+		final Element root = buildEqual( "html[1]", Root.class );
+
+		final List<Attribute> identifying = new ArrayList<>();
+		identifying.add( new StringAttribute( "id", "change-for-div" ) );
+		identifying.add( new StringAttribute( "text", "Change" ) );
+		identifying.add( new StringAttribute( IdentifyingAttributes.TYPE_ATTRIBUTE_KEY, "button" ) );
+		identifying.add( new PathAttribute( Path.fromString( "html[1]/body[1]/button[1]" ) ) );
+		identifying.add( new SuffixAttribute( 1 ) );
+		identifying.add( OutlineAttribute.create( new Rectangle( 0, 237, -1121, -237 ) ) );
+		identifying.add( OutlineAttribute.createAbsolute( new Rectangle( 8, 253, 63, 21 ) ) );
+
+		final MutableAttributes attributes = new MutableAttributes();
+		attributes.put( "shown", true );
+
+		final Element expected =
+				Element.create( "retestId", root, new IdentifyingAttributes( identifying ), attributes.immutable() );
+		root.addChildren( expected );
+
+		final Element actualRoot = buildEqual( "html[1]", Root.class );
+
+		final List<Attribute> actualIdentifying = new ArrayList<>();
+		identifying.add( new StringAttribute( "text",
+				"file:///Users/roessler/Documents/startup/workspace/recheck-web/src/test/resources/pages/non-existent.html" ) );
```

Comment:
Why this huge local path as attribute value? Wouldn't it be sufficient to use something simpler that differs from `"change-for-div"`?

Reply:
As of TDD, this is an exact replication of the problem. Why is that a problem?

Reply:
This is not a big deal, but it is noise IMHO. When we look at this in a few months, we will probably be confused.

The cause for the problem is not the huge local path. The test case name says "very different elements should not be aligned", so assume we also don't want "foo" and "barbaz4711" to not be aligned? Generalizing the bug may help us to get a better understanding.

It is up to you, we can also keep it as it is.